shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -293,11 +296,22 @@ View onCreateView(
   }
 
   void onActivityCreated(@Nullable Bundle bundle) {
-    Log.v(TAG, "onActivityCreated. Giving plugins an opportunity to restore state.");
+    Log.v(TAG, "onActivityCreated. Giving framework and plugins an opportunity to restore state.");
     ensureAlive();
 
+    Bundle pluginsBundle = null;
+    byte[] frameworkBundle = null;
```

Comment:
wouldn't call it bundle which is a Map<String, Parcelable> (more or less). Call it frameworkState or frameworkBundleData or something.